test/acceptance/behave/components/ngsiv2/entities/list_entities/list_entities.feature

```diff
@@ -0,0 +1,808 @@
+  As a context broker user
+  I would like to list all entities with get request and queries parameter using NGSI v2
+  So that I can manage and use them in my scripts
+
+  Actions Before the Feature:
+  Setup: update properties test file from "epg_contextBroker.txt" and sudo local "false"
+  Setup: update contextBroker config file
+  Setup: start ContextBroker
+  Check: verify contextBroker is installed successfully
+  Check: verify mongo is installed successfully
+
+  Actions After each Scenario:
+  Setup: delete database in mongo
+
+  Actions After the Feature:
+  Setup: stop ContextBroker
+
+  @happy_path
+  Scenario:  list all entities using NGSI v2 (3 entity groups with 11 entities in total)
+    Given  a definition of headers
+      | parameter          | value            |
+      | Fiware-Service     | test_happy_path  |
+      | Fiware-ServicePath | /test            |
+      | Content-Type       | application/json |
+    And initialize entity groups recorder
+    And properties to entities
+      | parameter         | value       |
+      | entities_type     | home        |
+      | entities_id       | room1       |
+      | attributes_number | 2           |
+      | attributes_name   | temperature |
+      | attributes_value  | 34          |
+      | attributes_type   | celsius     |
+      | metadatas_number  | 2           |
+      | metadatas_name    | very_hot    |
+      | metadatas_type    | alarm       |
+      | metadatas_value   | random=10   |
+    And create entity group with "5" entities in "normalized" mode
+      | entity | prefix |
+      | id     | true   |
+    And verify that receive several "Created" http code
+    And record entity group
+    And properties to entities
+      | parameter         | value                   |
+      | entities_type     | house                   |
+      | entities_id       | room2                   |
+      | attributes_number | 2                       |
+      | attributes_name   | timestamp               |
+      | attributes_value  | 017-06-17T07:21:24.238Z |
+      | attributes_type   | date                    |
+      | metadatas_number  | 2                       |
+      | metadatas_name    | very_hot                |
+      | metadatas_type    | alarm                   |
+      | metadatas_value   | random=10               |
+    And create entity group with "5" entities in "normalized" mode
+      | entity | prefix |
+      | id     | true   |
+    And verify that receive several "Created" http code
+    And record entity group
+    And properties to entities
+      | parameter        | value      |
+      | entities_type    | "car"      |
+      | entities_id      | "vehicle"  |
+      | attributes_name  | "speed"    |
+      | attributes_value | 89         |
+      | attributes_type  | "km_h"     |
+      | metadatas_name   | "very_hot" |
+      | metadatas_type   | "alarm"    |
+      | metadatas_value  | "hot"      |
+    And create an entity in raw and "normalized" modes
```

Comment:
I undertand you mean:

raw -> options=keValues
normalized -> no options (default mode)

In that case, why don't use?

And create an entity in keyValues and normalized modes